nengo/tests/test_builder.py

```diff
@@ -125,30 +124,6 @@ def test_signal_values():
         two_d.initial_value[...] = np.array([[0.5], [-0.5]])
 
 
-def test_signal_init_values(RefSimulator):
```

Comment:
FYI, for the next reviewer, this test is duplicated specifically by [this test](https://github.com/nengo/nengo/blob/master/nengo/tests/test_simulator.py#L140).